app/services/loader.js

```diff
@@ -48,6 +49,9 @@ export default Service.extend({
         } else {
           if (config.isFormData) {
             fetchOptions.body = objectToFormData(data);
+          } else if (config.isFile) {
+            fetchOptions.headers.Authorization = adapter.ajaxOptions().headers.Authorization;
```

Comment:
Headers already taken care of as long as you don't put `isExternal` in the options hash. So remove this please.